fvid/fvid.py

```diff
@@ -128,19 +203,63 @@ def save_bits_to_file(file_path, bits):
     ) as f:
         bitstring.tofile(f)
 
+def save_bits_to_file_crypto(file_path, bits, key):
+    bitstring_temp = Bits(bin=bits)
+    encrypted = bitstring_temp.tobytes()
+
+    # #zip
+    # in_ = io.BytesIO()
+    # in_.write(encrypted)
+    # in_.seek(0)
+    # with gzip.GzipFile(fileobj=in_, mode='rb') as fo:
+        # encrypted = fo.read()
+    # #zip
+
+    f = Fernet(key)
```

Comment:
@dobrosketchkun I am also experiencing the same thing as @Theelgirl 

[alfredo@DARTHNET test]$ fvid -i file.mp4 -d -p 
Enter password:
Reading video...
Bits are in place
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 275.97it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 283.82it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 304.40it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 301.18it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:04<00:00, 260.68it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 287.70it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 279.10it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:04<00:00, 265.74it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:04<00:00, 260.78it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 292.57it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 274.18it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 283.01it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:04<00:00, 263.16it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 289.46it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 290.54it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 299.10it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:02<00:00, 468.84it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 290.35it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:04<00:00, 266.55it/s]
Getting bits from frame: 100%|███████████████████████████████████████| 1080/1080 [00:03<00:00, 282.17it/s]
Traceback (most recent call last):
  File "/home/alfredo/.local/bin/fvid", line 8, in <module>
    sys.exit(main())
  File "/home/alfredo/.local/lib/python3.8/site-packages/fvid/fvid.py", line 367, in main
    save_bits_to_file_crypto(file_path, bits, key)
  File "/home/alfredo/.local/lib/python3.8/site-packages/fvid/fvid.py", line 220, in save_bits_to_file_crypto
    decrypted_bits = f.decrypt(encrypted)  # .decode()
  File "/usr/lib/python3.8/site-packages/cryptography/fernet.py", line 74, in decrypt
    timestamp, data = Fernet._get_unverified_token_data(token)
  File "/usr/lib/python3.8/site-packages/cryptography/fernet.py", line 92, in _get_unverified_token_data
    raise InvalidToken
cryptography.fernet.InvalidToken